src/accesspattern/datasubset.jl

```diff
@@ -1,7 +1,12 @@
 target(data) = target(identity, data)
 target(f, data) = f(data)
 
+
+
 target{N}(f, tuple::NTuple{N}) = target(f, tuple[N])
+target{N}(f, tuple::NTuple{N}, obsdim::NTuple{N}) = target(f, tuple[N], obsdim[N])
```

Comment:
Let's not restrict the type of `obsdim` here, since it is in general allowed to specify a single obsdim for a whole tuple